docroot/themes/custom/mass_theme/mass_theme.theme

```diff
@@ -4384,6 +4384,24 @@ function mass_theme_preprocess_field__person_org_role(&$variables) {
   $variables['person_org_roles'] = $person_roles;
 }
 
+/**
+ * Data to fetch the alerts as soon as possible.
+ *
+ * @see docroot/themes/custom/mass_theme/templates/layout/html.html.twig
+ */
+function mass_theme_build_data_from_alert_blocks(&$variables) {
+  $is_front_page = \Drupal::service('path.matcher')->isFrontPage();
+  /** @var Drupal\Core\Routing\CurrentRouteMatch */
+  $current_route_match_service = Drupal::service('current_route_match');
+  /** @var \Drupal\node\Entity\Node */
+  $node = $current_route_match_service->getParameter('node');
+  // Defining paths to fetch alerts.
+  $paths = [];
+  $paths[] = ['#path' => '/alerts/sitewide'];
+  !$is_front_page ? $paths[] = ['#path' => '/alerts/page/' . $node->id()] : NULL;
```

Comment:
@beto-aveiga I think this might be a problem for non-node pages, which will not have a node parameter to match.